lib/internal/modules/esm/default_resolve.js

```diff
@@ -53,7 +53,7 @@ function resolve(specifier, parentURL) {
   try {
     const parsed = new URL(specifier);
     if (parsed.protocol === 'data:') {
-      const [ , mime ] = /^([^/]+\/[^;,]+)(;base64)?,/.exec(parsed.pathname) || [ null, null, null ];
+      const [ , mime ] = /^([^/]+\/[^;,]+)(?:[^,]*?)(;base64)?,/.exec(parsed.pathname) || [ null, null, null ];
```

Comment:
Why this change? It seems unrelated to adding a new module.

Reply:
still rebasing, see last commit being `wip`... bad rebase